packages/zowe-explorer/src/dataset/ZoweDatasetNode.ts

```diff
@@ -326,7 +328,30 @@ export class ZoweDatasetNode extends ZoweTreeNode implements IZoweDatasetTreeNod
                 return sortByName(a, b);
             }
 
-            if (sort.method === DatasetSortOpts.LastModified) {
+            if (sort.method === DatasetSortOpts.DateCreated) {
```

Comment:
with the list growing for ways of sorting should we consider a switch/case in place of the continued if/else if's?